src/hierarchies/nnig_hierarchy.cpp

```diff
@@ -11,46 +11,58 @@
 
 void NNIGHierarchy::check_and_initialize() {
   check_hypers_validity();
-  mean = get_mu0();
-  sd = sqrt(get_beta0() / (get_alpha0() - 1));
+  mean = mu0;
+  sd = sqrt(beta0 / (alpha0 - 1));
```

Comment:
I left the setters, here and elsewhere, because priors may not have pointwise values of hyperparameters, thus requiring a mehtod to get one. Is this correct?

Reply:
About this. We always have "pointwise" values of hyperparameters. It is just a matter of whether they are fixed by the user (as of now) or the are updated at each iteration during the MCMC.
So there will always be some 'alpha0' the difference is that it might change in time.

Reply:
Great. I guess we can merge the PR then?

Reply:
Ok let's do it, then we can get back and change stuff it needed